validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/ExternalSigner.java

```diff
@@ -62,7 +63,12 @@ public ExternalSigner(
 
   @Override
   public SafeFuture<BLSSignature> createRandaoReveal(final UInt64 epoch, final ForkInfo forkInfo) {
-    return sign(signingRootForRandaoReveal(epoch, forkInfo), SignType.RANDAO_REVEAL, emptyMap());
+    return sign(
+        signingRootForRandaoReveal(epoch, forkInfo),
+        SignType.RANDAO_REVEAL,
+        Map.of(
+            "randao_reveal", Map.of("epoch", epoch),
+            "fork_info", forkInfo(forkInfo)));
```

Comment:
you can possibly use the constant `FORK_INFO` here as well.

Reply:
done